### DISCUSSION TITLE
How to access the goal_state from database using python interface ?

### DISCUSSION THREAD

**Discussion by chandarr** [ros-industrial/reach#79] [Q&A] [answered]
in the  heatmap_generator.py, the goal (positions) is accessed using r.goal()[0:3,3]. 

The other values of score and reachable are accessible using r.score and r.reachable. 

We want to use the goal_state values for each of the goals to perform certain analysis. 

Trying to use r.goal_state() generates a "no python class registered" error. 

How would you recommend to access the goal_state values ?

Thanks

**Comment by marip8** [comment, marked as answer]
Can you see if #80 fixes this issue?
